app/components/mandatenbeheer/mandataris/mandataris-table.hbs

```diff
@@ -0,0 +1,79 @@
+<AuDataTable
+  @content={{@content}}
+  @isLoading={{@isLoading}}
+  @noDataMessage="Geen mandatarissen gevonden"
+  @sort={{@sort}}
+  @page={{@page}}
+  @size={{@size}}
+  as |t|
+>
+  <t.content as |c|>
+    <c.header>
+      <th>Orgaan</th>
+      <AuDataTableThSortable
+        @field="isBestuurlijkeAliasVan.gebruikteVoornaam"
+        @currentSorting={{@sort}}
+        @label="Naam"
+      />
+      <AuDataTableThSortable
+        @field="isBestuurlijkeAliasVan.achternaam"
+        @currentSorting={{@sort}}
+        @label="Familienaam"
+      />
+      <AuDataTableThSortable
+        @field="heeftLidmaatschap.binnenFractie.naam"
+        @currentSorting={{@sort}}
+        @label="Fractie"
+      />
+      <AuDataTableThSortable
+        @field="bekleedt.bestuursfunctie.label"
+        @currentSorting={{@sort}}
+        @label="Mandaat"
+      />
+      <AuDataTableThSortable
+        @field="start"
+        @currentSorting={{@sort}}
+        @label="Start mandaat"
+      />
+      <AuDataTableThSortable
+        @field="einde"
+        @currentSorting={{@sort}}
+        @label="Einde mandaat"
+      />
+      <th></th>
+    </c.header>
+
+    <c.body as |row|>
+      <td>
+        {{#each
+          (get @mandatarisBestuursorganen row.id)
+          as |bestuursorgaan index|
+        }}
+          {{#if (gt index 0)}}<br />{{/if}}
```

Comment:
This seems a bit hacky, but ok

Reply:
Not my code, but should resolve itself in the future, with the new wireframes.